src/pages/home/HeaderView.js

```diff
@@ -85,8 +85,9 @@ const HeaderView = (props) => {
             };
         },
     );
+    const isPolicyRoom = ReportUtils.isPolicyRoom(props.report);
     const isDefaultChatRoom = ReportUtils.isDefaultRoom(props.report);
-    const title = isDefaultChatRoom
+    const title = isDefaultChatRoom || isPolicyRoom
```

Comment:
We need to show similar style of title for both, default and policy rooms. I need to keep them separate though as the default dooms have subtitle and avatar, which is a different behaviour and design than policy room.